src/pages/index.js

```diff
@@ -264,7 +256,10 @@ function getCardElement(data) {
         const nowLiked = cardLikeBtn.classList.contains(
           "card__like-button_liked"
         );
-        localStorage.setItem(`liked-${data._id}`, nowLiked);
+        const wasLiked = localStorage.getItem(`liked-${data._id}`) === "true";
```

Comment:
`localStorage` can't be used for liking because it works only in your own browser. If you visit your site on another computer, your likes won't be updated


You should check only `isLiked`